x-pack/plugins/security/server/__fixtures__/request.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import url from 'url';
+import { schema } from '@kbn/config-schema';
+import { KibanaRequest } from '../../../../../src/core/server';
+
+interface RequestFixtureOptions {
+  headers?: Record<string, string>;
+  params?: Record<string, unknown>;
+  path?: string;
+  search?: string;
+  payload?: unknown;
+}
+
+export function requestFixture({
+  headers = { accept: 'something/html' },
+  params,
+  path = '/wat',
+  search = '',
+  payload,
+}: RequestFixtureOptions = {}) {
```

Comment:
I added some mocks as a part of core, would you mind switching to them when they are merged? https://github.com/elastic/kibana/pull/40352

Reply:
Definitely!

Reply:
merged